package.json

```diff
@@ -23,14 +23,15 @@
   "scripts": {
     "build": "lerna run compile",
     "dev-build": "lerna run dev-compile --parallel --concurrency 10",
-    "postinstall": "lerna clean --yes && lerna bootstrap && node ./tools/init_dev.js -f",
+    "preinstall": "yarn workspace @pipcook/boa run install-dep",
+    "postinstall": "node ./tools/init_dev.js -f && rm -rf ./node_modules/midway-bin/node_modules/typescript/ && yarn workspace @pipcook/boa run compile",
     "docsify": "npm run typedoc && http-server -c-1 ./docs",
     "lint": "eslint . --ext .ts --ext .js && lerna run --scope @pipcook/daemon lint",
     "pretest": "npm run lint",
     "test": "sh run_tests.sh",
     "test:pipeline": "sh ./run_pipeline.sh",
     "typedoc": "sh tools/mkdoc.sh",
-    "clean": "lerna run clean",
+    "clean": "lerna run clean --parallel --concurrency 10",
```

Comment:
Shall we use yarn workspace here?

Reply:
Yarn workspace does not support parallel mode at this moment, therefore, to speedup those procedures that could run in parallel we should use `lerna`.

Reply:
How about using https://www.npmjs.com/package/concurrently?

Reply:
If we plan to totally get rid of `lerna`, we could add this. 
But for now, we still have some functionalities requires `lerna` (publishing & versioning). I think the best practice is to leave them here. And move to `concurrently` as the moment we do not need `lerna` anymore.

Reply:
But the `clean` is related to build script, and it looks a bit weird that we use yarn to build and lerna to clean files which are built by yarn.